fabric-resource-loader-v0/src/testmodClient/java/net/fabricmc/fabric/test/resource/loader/client/ResourceLoaderClientTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.resource.loader.client;
+
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.clickScreenButton;
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.openGameMenu;
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.submitServer;
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.takeScreenshot;
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.tapKey;
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.waitForClientTicks;
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.waitForLoadingComplete;
+import static net.fabricmc.fabric.test.base.client.FabricClientTestHelper.waitForScreen;
+
+import org.lwjgl.glfw.GLFW;
+
+import net.minecraft.block.Blocks;
+import net.minecraft.client.gui.screen.GameMenuScreen;
+import net.minecraft.client.gui.screen.option.OptionsScreen;
+import net.minecraft.client.gui.screen.pack.PackScreen;
+
+import net.fabricmc.fabric.test.base.client.ClientTest;
+import net.fabricmc.fabric.test.base.client.FabricClientTest;
+
+public class ResourceLoaderClientTest implements FabricClientTest {
+	@ClientTest
+	public void injectedResources() {
+		submitServer((world, player) ->
+				world.setBlockState(
+						player.getBlockPos().up().offset(player.getHorizontalFacing(), 1),
+						Blocks.DIAMOND_BLOCK.getDefaultState()
+				)
+		);
+		// Takes a little bit for our block to appear :D
+		waitForClientTicks(5);
```

Comment:
Looks like Github actions is even slower, need to find a good way to know when the chunk has been fully re-rendered, it is also needed after changing the resource packs 🤔